core/DataAccess/LogAggregator.php

```diff
@@ -1059,9 +1065,11 @@ protected function getActionsMetricFields()
      * @param bool|string $where An optional SQL expression used in the SQL's **WHERE** clause.
      * @param array $additionalSelects Additional SELECT fields that are not included in the group by
      *                                 clause. These can be aggregate expressions, eg, `SUM(somecol)`.
+     * @param RankingQuery|bool $rankingQuery
      * @return \Zend_Db_Statement
      */
-    public function queryConversionsByDimension($dimensions = array(), $where = false, $additionalSelects = array(), $extraFrom = [])
+    public function queryConversionsByDimension($dimensions = array(), $where = false, $additionalSelects = array(), $extraFrom = [],
+                                                $rankingQuery = false, $rankingQueryGenerate = false)
```

Comment:
The last parameter isn't documented. And from looking at the code it wasn't directly visible what a difference it makes. But seems when the ranking query is executed it directly returns all records where as otherwise it returns a query pointer, right?

Reply:
Yes that is correct, I'll add docs.